sparta/sparta/pipeViewer/Outputter.hpp

```diff
@@ -187,7 +187,7 @@ namespace sparta::pipeViewer
                     if(!dat.stringVector[i].empty()){
                         // We check if we have seen this exact pair, field and value before or not.
                         if(const auto& [val, str] = std::tie(dat.valueVector[i].first, dat.stringVector[i]);
-                           stringMap.try_emplace(std::make_tuple(dat.pairId, i, val), str).second) {
+                           stringMap.emplace(std::piecewise_construct, std::forward_as_tuple(dat.pairId, i, val), std::forward_as_tuple(str)).second) {
```

Comment:
Workaround for a Clang bug: https://github.com/llvm/llvm-project/issues/61415

This is essentially what `try_emplace` does under the hood, so performance shouldn't be affected.